core/src/banking_stage/forwarder.rs

```diff
@@ -228,11 +228,13 @@ impl Forwarder {
         usize,
         Option<Pubkey>,
     ) {
-        let (res, num_packets, forward_us, leader_pubkey) =
+        let (res, num_packets, _forward_us, leader_pubkey) =
             self.forward_packets(forward_option, forwardable_packets);
+        if let Err(ref err) = res {
+            warn!("failed to forward packets: {err}");
+        }
```

Comment:
This is no longer inside `if num_packet > 0`. Currently, we just return `Ok()` if `forwardable_packets` is empty (and thus return 0 for num packets forwarded). Seems slightly more defensive to have it outside, but I don't feel to strongly and shift it back inside if desired